test/check-application

```diff
@@ -51,13 +51,88 @@ class TestApplication(testlib.MachineCase):
+
+        # switch to all containers
+        b.wait_present("#containers-containers-filter button")
+        b.click("#containers-containers-filter button")
+        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='all'] a")
+        b.wait_present("#containers-containers")
+        b.wait_present('#containers-containers tr:contains("busybox:latest") td:contains("running")')
+        b.wait_present('#containers-containers tr:contains("alpine:latest")')
+
+        # delete a stopped container
+        b.click('#containers-containers tbody tr:contains("alpine:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        with b.wait_timeout(60):
+            b.wait_present(".modal-dialog div")
+        b.wait_visible(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        with b.wait_timeout(60):
+            try:
+                b.wait_visible("#containers-containers")
+                b.wait_not_in_text("#containers-containers", "alpine:latest")
+            except:
+                self.confirm()
+                b.wait_visible("#containers")
+                b.wait_not_in_text("#containers-containers", "alpine:latest")
+
+        # delete a running container
+        self.loadAllContainerlist()
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        with b.wait_timeout(60):
+            b.wait_present(".modal-dialog div")
+        b.wait_visible(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        with b.wait_timeout(60):
+                self.confirm()
+                with b.wait_timeout(60):
+                    b.wait_not_in_text("#containers-containers", "busybox:latest")
+
+        # run a container for Stop test
+        m.execute("podman run -d --name swamped-crate busybox sleep 1000")
+        self.loadAllContainerlist()
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
+
+        # stop a running container
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr:contains("Stop")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr:contains("Stop")')
+        self.loadAllContainerlist()
```

Comment:
This should not need a page reload. If you commonly use that case, split the function into two: one for an explicit "page reload and wait for it to be built up again", and one for "load and expand the container list with all containers".